source/globalCommands.py

```diff
@@ -1956,6 +1961,57 @@ def script_reportOrShowFormattingAtCaret(self, gesture):
 		elif repeats == 1:
 			self.script_showFormattingAtCaret(gesture)
 
+	@script(
+		description=_(
+			# Translators: the description for the reportDetailsSummary script.
+			"Report summary of any annotation details at the current location."
+		)
+	)
+	def script_reportDetailsSummary(self, gesture):
+		"""Report the annotation details summary for the single character under the caret or the current nav object.
+		@note: It is tempting to try to report any annotation details that exists in the range formed by prior
+			and current location. This would be a new paradigm in NVDA, and may feel natural when moving by line
+			to be able to more quickly have the 'details' reported. However, there may be more than one 'details
+			relation' in that range and, we don't yet have a way for the user to select which one to report.

```

Comment:
I'm not a native English speaker but the comma after 'and' looks strange to me.

Reply:
Yes, good catch.